app/components/index-scroll-container.js

```diff
@@ -1,15 +1,25 @@
 import Ember from 'ember';
-import { vars } from 'site/lib/vudu';
+import v from 'npm:vudu';
+import c, { vars } from 'site/lib/vudu';
 
 const {
   get,
   Component,
   inject: { service },
 } = Ember;
 
+const styles = v({
+  indexScrollContainer: {
+    backgroundColor: c.lightGray.color,
+    '@composes': [c.col11, c.mxAuto],
+  }
+});
+
 const INDEX_PAGE_ID = '#index-route-scroll-context';
 
 export default Component.extend({
+  classNames: [styles.indexScrollContainer],
```

Comment:
all component classnames should end in `Component` for sake of reading the dom properly